### PR TITLE
feat(website): apply design

### PR DESCRIPTION
**Pull request by azu**
Roughly, I tried to apply a design to website that is based on @uetchy's design  #467 

- https://scene.zeplin.io/project/5a5d503f98c6ac212656f7fe

## Preview

[image](https://deploy-preview-480--textlint.netlify.com/)

You can preview that on <https://deploy-preview-480--textlint.netlify.com/>

- <https://deploy-preview-480--textlint.netlify.com/>

## Discussion Points

I've found some discussion points

### Docusaurus's `primaryColor` sync to `<a>` link color

It is hard to see.

image

### Gradient header

Docusaurus's header is fixed.
Is this header design OK?

### Additional contents

@0x6b suggestion that

> Quick installation and command–line linting session using asciinema comes to my mind (just an idea) e.g. https://asciinema.org/a/378

### Mobile

image

Also, Playground will be disabled in mobile.
It prevent smooth scroll on mobile.

### CODE REVIEW

File: website/static/css/index.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    border-radius: 100px;
+    background-color: var(--textColor);
+    font-family: Lato;
+    font-size: 16px;
+    font-weight: bold;
+    text-align: center;
+    border: none;
+}
+
+.GettingStartedButton:hover {
```
Comment (azu) [inline website/static/css/index.css]: I've just use reverse color as hover coloring.

**Comment by 0x6b** [comment]
> Quick installation and command–line linting session

I created an animated GIF as proof of concept. Not cool as asciinema since no color, and too big for landing page, but you can see what it looks like.

install-and-lint

**Comment by azu** [comment]
📝  https://github.com/prettier/prettier/pull/3718
[Prettier's new design](https://deploy-preview-3718--prettier.netlify.com/) just write "Get Started" directly.

It is just an idea.

prettier opinionated code formatter 2018-01-18 01-10-22

Align text and image description.

| Text Desctipn | Image Description |
| ----  | ---- |
| Text  | Image |
| Text  | Image |
| Text  | Image |

**Comment by azu** [comment]
I think that Docusaurus require high-brightness color as `primaryColor`.
I try to change `primaryColor` to `#29bfcd`.
This color brightness `50`.
image

**Comment by azu** [comment]
- [ ] mobile width is broken

**Comment by azu** [comment]
I've introduce transition header.
This idea is inspired by https://elevenback.jp/

**Comment by azu** [comment]
https://webaim.org/resources/contrastchecker/?fcolor=29BFCD&bcolor=F9F9F9
Current contrast is not accessible.
We need to make accessible design.

**Comment by uetchy** [comment]
How about `#155F66` and `#FFFFFF`?

**Comment by azu** [comment]
@uetchy I've changed primaryColor to `#155F66` as a trial.

https://deploy-preview-480--textlint.netlify.com/

image

image

**Comment by azu** [comment]
I'll merge this after tweak color and mobile optimization.

**Comment by azu** [comment]
I've changed color by pages.
image
image
image

**Comment by azu** [comment]
Add short getting started.
image

**Comment by azu** [comment]
Merge 🎉 
I'll take next task https://github.com/textlint/textlint/issues/465

Thanks again @0x6b @uetchy

**Pull request merged**